qacode/core/utils/Utils.py

```diff
@@ -27,13 +28,14 @@ def get_file(version_py=version_info.major,file_path=None, file_name=None, encod
         if version_py == 2:
             with open(file_path_join) as f:
                 return f.read().decode(encoding)
-
+    @staticmethod
     def get_file_abspath(file=None):
```

Comment:
Sorry, don't mean to be harsh but, why this method even exists?

It's just calling `path.abspath(path.dirname(method_only_parameter))`, it doesn't appear to provide something useful compare to just calling those methods, also it doesn't work for directories that don't exists right now (a valid use case in my opinion, what if I want to create it right now and prefer to use the absolute path? I can't use your method).

As a side note, never, **ever** use `file` as a variable or parameter name in Python, you're shadowing the native [`file`](https://docs.python.org/2/library/stdtypes.html#file-objects) type in Python 2 (yeah, a bad name for a native type, maybe that's why it was removed from Python 3 😆 ).